src/test/java/dev/blaauwendraad/masker/json/JsonPathArraysLookupTest.java

```diff
@@ -0,0 +1,21 @@
+package dev.blaauwendraad.masker.json;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.IOException;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JsonPathArraysLookupTest {
+    @ParameterizedTest
+    @MethodSource("jsonPathFile")
+    void test(JsonMaskerTestInstance testInstance) {
```

Comment:
In general you don't have to name test instances something with test or even postfix them with `test` because it is already annotated with `@ParameterizedTest` and doesn't give much info while running. Something like `jsonPath` is slightly nicer

Reply:
I agree that "test" is implied from the context. I want to keep the method name a verb though, so I will come up with something in-between.